src/jolly_brancher/main.py

```diff
@@ -32,16 +33,37 @@
 from jolly_brancher import __version__
 
 # DONOTCOMMIT
-REPO_ROOT = "/home/ahonnecke/src"
 REMOTE = "upstream"
-TOKEN = ""
 
 __author__ = "Ashton Von Honnecke"
 __copyright__ = "Ashton Von Honnecke"
 __license__ = "MIT"
 
 _logger = logging.getLogger(__name__)
 
+# CONFIG VARS
+KEYS_AND_PROMPTS = [['auth_email', 'your login email for Atlassian'], ['base_url', 'the base URL for Atlassian (e.g., https://cirrusv2x.atlassian.net)'], ['token', 'your Atlassian API token which can be generated here (https://id.atlassian.com/manage-profile/security/api-tokens)'], ['repo_root', 'the path to the root directory for the repository']]
+CONFIG_FILENAME = 'example'
+DEFAULT_SECTION_NAME = 'DEFAULT'
+
+def config_setup():
+    os.chdir('../..')
```

Comment:
This should probably be absolute, I would guess something like `~/.config/jolly_rancher.ini`, but we can iterate on that!